helpers/helper.js

```diff
@@ -0,0 +1,24 @@
+const multer = require('multer');
+
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, 'uploads');
+    },
+    filename: (req, file, cb) => {
+        cb(null, new Date().toISOString().replace(/:/g, '-') + '-' + file.originalname);
+    }
+});
+const filefilter = (req, file, cb) => {
+    if (file.mimetype === 'image/png' || file.mimetype === 'image/jpg' 
+        || file.mimetype === 'image/jpeg'){
+            cb(null, true);
+        }else {
+            cb(null, false);
+        }
+}
+
+const upload = multer({storage: storage, fileFilter: filefilter});
+
+

```

Comment:
why too many blank space here and why file is not formatted?

Reply:
removed blank space